### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I:
✔️ Used [the given template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
✔️ Created a database named vet_clinic.
✔️ Created a table named animals with the columns: id, name, date_of_birth, escape_attempts, neutered, weight_kg
✔️ Copied the SQL into the schema.sql file in the generated repository.
✔️ Inserted animals' data:
✔️ Copied the SQL into the data.sql file in the generated repository.
✔️ Wrote queries to quest the database
✔️ Took a screenshot of the queries' results.
✔️ Copied the SQL into the queries.sql file in the generated repository.
✔️ Added the screenshots with the results of the queries to my Pull Request description.

image

image

image

### CODE REVIEW

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @diarisdiakite, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed

## Required Changes ♻️
- [ ] Nice work using the recommended Microverse [README template](https://github.com/microverseinc/readme-template) for this project. However, it left a few changes that needs to be considered when adopting this template:
  - Personalize the project name and project summary to the name of your project and a summary of what you implemented. :+1:
  - Remove all placeholder texts that describe what should be done in each section sunglasses
  - In the _**Tech stack section of the README**_, kindly add the appropriate technologies in the Client, Server and Database subsections. Those sections contain technologies that were not used in this project. :+1: :sunglasses:
  - In the _**Key Features section**_, kindly add 1-3 things that you accomplished in this project. I would also advise you to get used to adding these features because you would do that for every project in Microverse and possibly in the professional world. :+1:
  - In the _**Setup subsection**_, kindly include ways to clone this project and change the directory to the directory of the project.
  - The _**Usage subsection**_ should contain ways to run the project in the browser if while the Run tests subsections should contain the commands for running the tests written in the project. :+1: :sunglasses:
  - The _**Authors**_ section should contain the social media handles of the owner of the repository and all other collaborators
  - Make sure the license section is not broken

- [ ] Nice work creating queries to answer the questions :clap: However, it requested in the project requirement to add screenshots of the query results to the pull request description. Kindly add the query screenshots to the PR to follow the project requirement.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -65,15 +27,13 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 [your_project_name] <a name="about-project"></a>
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] You have done incredibly well almost implementing all required changes but a key part is still missing, and that is the project name which is more like a hyperlink here. Kindly customize this to reflect the appropriate title of your project and to also ensure that users understand or feel what your project is about. Here's where the previous reviewer requested it:
image

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @diarisdiakite :wave: :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so_close


 ## Highlights
- Good PR message ✅
- Good job with the screenshots :heavy_check_mark: 
- Great commit messages ✅ 
- No linter error ✅ 
- Professional ReadMe template :heavy_check_mark:

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @diarisdiakite, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Change :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 

## Optional suggestions
- [ ] Consider removing the `Live Demo` section from your readme file as you have no live link to share for this kind of project

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**